test/Libraries/RevitTestServices/RevitSystemTestBase.cs

```diff
@@ -36,17 +142,14 @@ public void Shutdown()
     }
 
     [TestFixture]
-    public abstract class RevitSystemTestBase : SystemTestBase
+    public class RevitSystemTestBase : SystemTestBase
     {
-        #region public methods
+        private string samplesPath;
+        private string defsPath;
+        protected string emptyModelPath1;
+        protected string emptyModelPath;
 
-        [SetUp]
-        public override void Setup()
-        {
-            base.Setup();
-
-            DocumentManager.Instance.CurrentUIApplication.ViewActivating += CurrentUIApplication_ViewActivating;
```

Comment:
I didn't remove this hook completely. I just moved it into SetupCore. See below. Once I moved the ViewActivating hook, the Setup override was no longer needed.